cmake/DAGMCConfig.cmake.in

```diff
@@ -17,3 +17,5 @@ message(STATUS "Configuring DAGMC ${DAGMC_VERSION}")
 set(DAGMC_INCLUDE_DIRS @CMAKE_INSTALL_PREFIX@/@INSTALL_INCLUDE_DIR@ @MOAB_INCLUDE_DIRS@)
 set(DAGMC_LIBRARY_DIRS @CMAKE_INSTALL_PREFIX@/@INSTALL_LIB_DIR@ @MOAB_LIBRARY_DIRS@)
 set(DAGMC_LIBRARIES @DAGMC_LIBRARIES@)
+
+include(@CMAKE_INSTALL_PREFIX@/lib/cmake/DAGMCTargets.cmake)
```

Comment:
Can you make sure this file ends with a newline?

Reply:
Can do!

Reply:
Curious though.. why?

Reply:
Best practice to have all files end with a newline